data/sql/updates/pending_db_world/rev_1550101831834633300.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1550101831834633300');
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1550098994179305900');
+-- Fallen Hero of the Horde 
+-- Fixes gossip text and completion of quest
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=14 AND `SourceGroup`=840;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(14,840,1451,0,1,9,0,2801,0,0,0,0,0,"","Show gossip text 1451 if player has quest 'A Tale of Sorrow' taken");
+
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (841, 842);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES
+(841,1392),
+(842,1411);
+
+UPDATE `gossip_menu_option` SET `OptionText`="Why are you here?", `OptionBroadcastTextID`=3582, `ActionMenuID`=841 WHERE `MenuID`=840 AND `OptionID`=0;
+DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (841, 842);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
+(841,0,0,"Continue with your story.",3601,1,1,842,0,0,0,"",0,0),
+(842,0,0,"Tragic...",3830,1,1,0,0,0,0,"",0,0);
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=7572;
+INSERT INTO `smart_scripts` VALUES (7572, 0, 0, 2, 62, 0, 100, 1, 842, 0, 0, 0, 0, 26, 2784, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Quest Credit \'Fall From Grace\'');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 1, 3, 62, 0, 100, 1, 881, 0, 0, 0, 0, 26, 2801, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Quest Credit \'A Tale of Sorrow\'');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 0 Selected - Close Gossip');
+INSERT INTO `smart_scripts` VALUES (7572, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fallen Hero of the Horde - On Gossip Option 1 Selected - Close Gossip'); 
```

Comment:
Add a new line at the end of the file, please.

Reply:
There you go, why is that needed?

Reply:
Consistency.